tests/upgrades/test_role.py

```diff
@@ -207,6 +226,8 @@ class TestRoleAddPermissionWithFilter:
 
     :expectedresults: The added permission with filter in existing
         'Default role' is intact post upgrade
+
+    :CaseAutomation: NotAutomated
```

Comment:
Here it is Automated, right?

Reply:
I removed it as it's redundant here as on module level is set to `:CaseAutomation: NotAutomated`.